common-service-core/src/test/java/com/ca/mfaas/eurekaservice/client/util/EurekaMetadataParserTest.java

```diff
@@ -103,4 +109,94 @@ public void testParseToListRoute() {
         assertEquals("List route size is different", 3, actualRoutes.size());
         assertThat(actualRoutes, containsInAnyOrder(expectedListRoute.toArray()));
     }
+    @Test
+    public void generateFullMetadata() {
+        String serviceId = "test service";
+        String gatewayUrl = "api/v1";
+        String version = "1.0.0";
+        String swaggerUrl = "https://service/api-doc";
+        String documentationUrl = "https://www.zowe.org";
+        String metadataPrefix = API_INFO + ".api-v1.";
+
+        ApiInfo apiInfo = new ApiInfo("org.zowe", gatewayUrl, version, swaggerUrl, documentationUrl);
+        try {
+            Map<String, String> metadata = eurekaMetadataParser.generateMetadata(serviceId, apiInfo);
```

Comment:
The static method generateMetadata(String, ApiInfo) from the type EurekaMetadataParser should be accessed in a static way
`Map<String, String> metadata = EurekaMetadataParser.generateMetadata(serviceId, apiInfo);`